src/pages/CoinList/index.js

```diff
@@ -75,13 +68,28 @@ class CoinList extends React.Component {
           hasError: false,
         });
       } else {
+        const storageFavoriteList = JSON.parse(
+          localStorage.getItem("favoriteList")
+        );
+        if (!storageFavoriteList) {
+          return;
+        }
+        if (Object.values(storageFavoriteList).length === 0) return;
+        const favoriteList = Object.values(storageFavoriteList).reduce(
+          (acc, current, index, array) => {
+            if (index === array.length - 1) {
+              return acc + `${current}`;
+            }
+            return acc + `${current}%2C%20`;
+          },
+          ""
+        );
```

Comment:
you cannot use query-string to parse this?

Reply:
I tryed it and I cant get it to return the proper format